src/wall_segment_sim.cpp

```diff
@@ -94,13 +94,11 @@ bool WallSegmentSim::Trace(double x, double y, double theta, double length, doub
         hit_chance += hit.second->angular_return * angle_of_incidence* 180.0/M_PI;  // Convert 
         hit_chance = constrain(hit_chance, hit.second->min_return, hit.second->max_return);
       }
-
       if (uniform_dist(random) < hit_chance) {  // If the light hit
-        // ROS_INFO_STREAM("Closest: " << range);
         range = hit.first.fraction * length;
         return true;
       } else {  // We didn't 'hit', pass through if transparent or no return if opaque
-        if (hit.second->type == "opaque") {
+        if (hit.second->type == "opaque" || hit.second->type == "reflective") {
```

Comment:
aren't opaque and reflective kinda the same?

Reply:
okay So I am gonna have two names “wall” and “reflective tape” and both type will be “opaque”